### PR TITLE
Document focus and sleep caveats in Controllers, gamepads and joysticks

### PR DESCRIPTION
**Pull request by Calinou**
- Improve line wrapping for long lines in scripts.
- Remove outdated paragraphs about Godot 3.3 changes.

### CODE REVIEW

File: tutorials/inputs/controllers_gamepads_joysticks.rst [hunk 1]
```diff
@@ -57,8 +57,10 @@ There are 3 ways to get input in an analog-aware way:
     # The line below is similar to `get_vector()`, except that it handles
     # the deadzone in a less optimal way. The resulting deadzone will have
     # a square-ish shape when it should ideally have a circular shape.
-    var velocity = Vector2(Input.get_action_strength("move_right") - Input.get_action_strength("move_left"),
-		Input.get_action_strength("move_back") - Input.get_action_strength("move_forward")).clamped(1)
+    var velocity = Vector2(
+            Input.get_action_strength("move_right") - Input.get_action_strength("move_left"),
+            Input.get_action_strength("move_back") - Input.get_action_strength("move_forward")
+    ).clamped(1.0)
```
Comment (skyace65) [inline tutorials/inputs/controllers_gamepads_joysticks.rst:63]:
```suggestion
    ).limit_length(1.0)
```

Clamped was deprecated in 3.x and the class reference recommended using limit_length instead

File: tutorials/inputs/controllers_gamepads_joysticks.rst [hunk 2]
```diff
@@ -70,8 +72,10 @@ There are 3 ways to get input in an analog-aware way:
     // The line below is similar to `get_vector()`, except that it handles
     // the deadzone in a less optimal way. The resulting deadzone will have
     // a square-ish shape when it should ideally have a circular shape.
-    Vector2 velocity = new Vector2(Input.GetActionStrength("move_right") - Input.GetActionStrength("move_left"),
-		Input.GetActionStrength("move_back") - Input.GetActionStrength("move_forward")).Clamped(1);
+    Vector2 velocity = new Vector2(
+            Input.GetActionStrength("move_right") - Input.GetActionStrength("move_left"),
+            Input.GetActionStrength("move_back") - Input.GetActionStrength("move_forward")
+    ).Clamped(1.0);
```
Comment (skyace65) [inline tutorials/inputs/controllers_gamepads_joysticks.rst:78]:
```suggestion
    ).limit_length(1.0);
```

**Review by skyace65 — CHANGES_REQUESTED** [review]

**Review by skyace65 — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Valuable information here, thanks! Merging.

**Comment by mhilbrunner** [comment]
Pushed to `4.0` and `stable`.

**Pull request merged**